templates/invite_contributors.html

```diff
@@ -0,0 +1,65 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Invitation</title>
+    <style>
+        .container {
+            padding: 20px;
+        }
+
+        .divider {
+            width: 58%;
+            border-top: 1px solid #ccc;
+        }
+        .button {
+          padding: 10px 20px;
+          font-weight: bold;
+          background-color: #007bff;
+          color: #fff;
+          text-decoration: none;
+            border-radius: 5px;
+        }
+        .iconify {
+            width: 7rem;
+            height: 7rem;
+            color: rgba(0, 123, 255, 0.92);
+        }
+        .accept-top{
+            padding-bottom: 2rem;
+        }
+        .accept-bottom{
+            padding-top: 2rem;
+            padding-bottom: 3rem;
+        }
+        p{
+            font-family: sans-serif;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="container">
+        <div>
+            <span class="iconify" data-icon="line-md:email"></span>
+        </div>
+    <div class="divider"></div>
+    <h1>Hello {{ ' ' ~ first_name if first_name else '' }}{{ ' ' ~ last_name if first_name else '' }}!</h1>
+
+    <div>
+        <p>We are pleased to grant you {{ 'a' if role == "viewer" else "an"}} {{ role }} access to {{ study_name }}!</p>
+        <p>Your authorization enables you to delve into this specialized
+            study area and explore its curated resources.</p>
+        <p class="accept-top">Please click the button to accept the invitation.</p>
+        <a href="{{ accept_url }}" class="button">View Study</a>
```

Comment:
The `accept_url` should also be provided as text as well. Maybe another line underneath so users can copy and paste